pandora-tests/test/utils/server.ts

```diff
@@ -0,0 +1,141 @@
+import { Assert, AssertNotNullable } from './utils';
+import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
+import { TEST_DIRECTORY_PORT, TEST_SERVER_DIRECTORY_ENTRYPOINT, TEST_SERVER_DIRECTORY_TEST_DIR, TEST_TEMP } from '../_setup/config';
+import { test } from '@playwright/test';
+import path from 'path';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+interface DirectoryEnvSetup {
```

Comment:
is there any way to not have a duplicate of this, like what if someone adds a new config to directory but forgets to add it here?

Reply:
Created an issue #376 for this